docs/conops.md

```diff
@@ -0,0 +1,146 @@
+
+The innovation can come from hardware (advanced manufacturing, superior procurement management, economic design, etc), or from algorithm. Some well-funded players may not care about recouping CapEx, so they would develop algorithms to analyze the supply and demand of the market, pricing of other services, etc., to programmatically price their trips.
+
+#### Pricing as a Service
+Arrow may offer a set of APIs for OEMs to develop in-house algorithms, but it could also be a product offering.
+
+Since Arrow has the full-access to everything on the network, it may makes sense to offer a white-labeled pricing algorithm solution to OEM players, which 1) provides an additional source of revenue to Arrow, and 2) help these OEM players to narrow their focus to the things that they are good at -- making vehicles.
+
+We believe this business model creates a win-win situation where it releases the production and legal burdens from Arrow, and it serves as a direct-to-consumer channel to OEMs, allowing them to monetize on their vehicles with a shorter time-to-market than if they have to sell them through contracts.
+
+
+#### Downside
+While the marketplace business model bridges many gaps on the market amount different participants, the platform (Arrow) itself may under-perform as operating a marketplace is difficult and the success is largely a function [network effects](https://online.hbs.edu/blog/post/what-are-network-effects). 
+
+There essentially three parties in a marketplace ecosystem: the platform (Arrow), the service providers (OEMs or other fleet owner-operators), the customers (passengers, corporations, etc.).
+
+At the early stage, Arrow would have to make outreach efforts to both service providers and customers because a lack of either parties will fail the business. If Arrow were to pursue this business model, a well-thought-out go-to-market strategy is needed and the execution has to be near flawless.
+
+#### Further Discussions
+Key points to discuss:
+* Whether Arrow wants to pursue a marketplace model
+* If we go with the vertical integration model, what kind of operations are we capable of running commercially
+* Who and when to showcase our demo
+* Should we seek strategic investors
+
+
+## Risks and Potential Issues
+***The underlying pricing model may be incorrectly constructed, thereby over- or under-price trips.***
+
+The pricing logic of `svc-pricing` derives from our [pricing model](https://docs.google.com/spreadsheets/d/1mjPtaIn3E5m7r4nyKt_sJKG9BSFm2ty7Gzo7OqERxwo), which was constructed without the assistance and review from external consultants. And currently, the pricing model makes a number of rough estimates on some key inputs like the cost basis of a cargo aircraft. With inaccurate pricing Arrow's business, reputation, results of operations and financial condition can be materially and adversely affected. 
```

Comment:
Rephrase "constructed without assistance and review" to say "our pricing model is preliminary and will be submitted for review to external consultants", because we should/will. Not our intention to never let others critique it

Reply:
[00d00e5](https://github.com/Arrow-air/svc-pricing/pull/3/commits/00d00e5af21d9fe993cefdae5033554b1ab6a211)